parser/src/powdr.lalrpop

```diff
@@ -81,9 +81,10 @@ ConstantDefinition: PilStatement<T> = {
 }
 
 PolynomialDefinition: PilStatement<T> = {
-    <@L> PolCol <Identifier> "=" <Expression> => PilStatement::PolynomialDefinition(<>)
+    <loc:@L> PolCol <id:Identifier> "=" <expr:Expression> => PilStatement::PolynomialDefinition(<>)
```

Comment:
is this change really required?

Reply:
Good catch, it was left over from experimentation